src/BoundsInference.cpp

```diff
@@ -1124,15 +1129,12 @@ class BoundsInference : public IRMutator {
                 }
             }
 
-            if (fused_with_f.empty()) {
+            if (fused_with_f.size() == 1) {
```

Comment:
Could you explain this change?

Reply:
Previously, ``` fused_with_f ``` contained a list of stages fused with the current, but not the current stage itself. In this PR, I changed it to contain current stage as well, because it seems to simplify some of the processing. Also, I guess the name of the set is not accurate anymore (unless we consider f to be fused with itself, I guess), so I renamed it to just ``` fused_group ```.